pkg/cmd/root/root.go

```diff
@@ -89,9 +92,11 @@ func NewCmd() *cobra.Command {
 	cmd.PersistentFlags().BoolP(gf.Imdsv2Flag, "I", false, "whether to enable IMDSv2 only, requiring a session token when submitting requests (default: false, meaning both IMDS v1 and v2 are enabled)")
 	cmd.PersistentFlags().Int64(gf.RebalanceDelayInSecFlag, 0, "rebalance rec delay in seconds, relative to the application start time (default: 0 seconds)")
 	cmd.PersistentFlags().String(gf.RebalanceTriggerTimeFlag, "", "rebalance rec trigger time in RFC3339 format. This takes priority over "+gf.RebalanceDelayInSecFlag+" (default: none)")
+	cmd.PersistentFlags().Int64P(gf.ASGTerminationDelayInSecFlag, "g", 0, "asg termination delay in seconds, relative to the application start time (default: 0 seconds)")
+	cmd.PersistentFlags().Int64P(gf.ASGTerminationTriggerTimeFlag, "", 0, "asg termination trigger time in RFC3339 format. This takes priority over "+gf.ASGTerminationDelayInSecFlag+" (default: none)")
 
 	// add subcommands
-	cmd.AddCommand(spot.Command, events.Command)
+	cmd.AddCommand(spot.Command, events.Command, asglifecycle.Command)
```

Comment:
can we have `asgLifecycleTermination` throughout to maintain consistency excluding the places were we define the kind to be asglifecycle?

Reply:
in this line you highlighted, the asglifecycle is of kind asglifecycle. I can't rename it without renaming all packages to asglifecycletermination. I'll keep this one as is if that's okay?

Reply:
yeah you can keep it, and modify other places wherever it is possible, thanks

Reply:
Resolution: keeping everything as asgLifecycle until Launch is introduced. When launch is introduced, will rename